### ISSUE TITLE
[Screen Reader-Command Palette]-Narrator focus moves separately onto the shortcut text which is appended to the command.

### ISSUE THREAD

**Issue by DasaradhaKamjula** [microsoft/terminal#7913]
**User Experience:**
Users who navigates in scan mode will get redundant info if focus moves separately onto shortcut

**Test Environment:**
App: Windows Terminal Preview
Feature: Command Palette
Screen Reader: Narrator
OS: Win 10 2004 Build (19041.508)

**Repro Steps:**
1.Open Windows terminal Preview app
2.Open Command Palette using shortcut (Ctrl+shift+p)
3.Search for any command in the search field
4.Turn on Scan mode, Navigate using arrow key in the suggestion list.
5.Verify whether narrator focus moves onto shortcut text separately or not

**Actual:**
Narrator focus moves separately onto the shortcut text which is appended to the command due to this screen reader user will get short cut info twice and that will be confusing

**Expected:**
Narrator focus should move onto entire command at once only, no need to move separately as that shortcut info is already announced when the focus is on entire command.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b24201d1-2276-4471-8183-e1e5353031ad}

**Comment by DHowett** [comment]
This fix will be deployed to the stable channel tomorrow (Thursday)

**Comment by zadjii-msft** [comment]
@Chandresh1007, @DasaradhaKamjula, @Priyanka-Chauhan123 - can you confirm that this is fixed, and close it if it is? This should have shipped in 1.6. Thanks!

**Comment by Priyanka-Chauhan123** [comment]
Issue is fixed. Verified on below environment. 
Windows Terminal App version: 1.9.1942.0. 
OS: Windows 10 build 21390.2025.
Hence, closing the bug.

**Comment by Priyanka-Chauhan123** [comment]
Closing the bug as issue has been fixed.
